docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -140,9 +140,11 @@ code: |
   else:
     school.recipient.name = school.special_education_director_or_coordinator.name
     school.recipient.address = school.special_education_director_or_coordinator.address
-    if not knows_special_education_director_or_coordinator_name:
-      school.special_education_coordinator.name = "Special Education Coordinator"
-  school.recipient.name
+  if knows_special_education_director_or_coordinator_name:  
+    school.special_education_director_or_coordinator.name.first
```

Comment:
Instead of putting both `school.special_education_director_or_coordinator.name.first` and `school.special_education_director_or_coordinator.name.last` in this interview order, you can just put `school.special_education_director_or_coordinator.name` and it will call the screen that they are both on